src/content/docs/tutorial-customer-experience/improve-customer-experience.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: "Improve your customer experience" 
+metaDescription: "New Relic observability maturity series: this guide gives you a quality foundation for optimizing your web application's performance to improve your customer experience."
+redirects:
+---
+
+After you've [established your current state](/docs/tutorial-customer-experience/establish-current-state/), it's time to take the information you've gathered and use it to start improving your customer experience. Since needs can vary based on each organization, you may have different areas you need to improve. The following steps are a suggested order to use for improving your customers' experience, and you may find that you can skip some steps based on your KPIs. As this is a continuing process, we recommend you bookmark this page so that you can refer to it in the future as your KPIs change. 
+
+<Steps>
+    <Step>
+### Decide which KPIs to improve
+
+After you [establish your current state](/docs/tutorial-customer-experience/establish-current-state/), you know what your user experience looks like across your system. From there, you need to determine which of the following areas need to improve:
+
+1. **Start with business priorities:** If you have clear business directives, you should focus on what matters most to your organization. For example, if your company has recently launched a new initiative around a line of business but the KPIs associated with the UI are below target, this is a great place to focus your efforts.
+2. **Focus on your KPIs:** Your key performance indicators are called "key" for a reason! They provide hard data on the various aspects of your system that act as pain points for your customers.
```

Comment:
I'd rewrite 2 as "**Align your data to your KPIs**" and then focus on how this can be done. Also, I don't think we need to remind people why KPIs are "key" :)

Reply:
Perhaps I was a bit to cheeky. Thanks!